app/templates/components/preprint-form-subjects.hbs

```diff
@@ -17,42 +17,48 @@
 
 <div class="row">
     <div class="col-md-4">
-        <ul>
+        <ul style="overflow: scroll">
             {{input value=filter.[0].value class="form-control" placeholder="Search"}}
             {{#each sortedTaxonomies as |element|}}
-                <li {{action 'select' element}} class={{if (eq path.[0].name element.name) 'selected'}}>
-                    {{element.name}}
-                </li>
+                {{#if (if-filter element.name filter.[0].value)}}
+                    <li {{action 'select' element}} class={{if (eq path.[0].name element.name) 'selected'}}>
+                        {{element.name}}
+                    </li>
+                {{/if}}
             {{/each}}
         </ul>
     </div>
 
     <div class="col-md-4">
-        <ul>
+        <ul style="overflow: scroll">
```

Comment:
When possible, and as time permits, we encourage styles to go in a `componentName.scss` file, rather than in templates.